server/api_server/api_server_start.py

```
@@ -335,47 +483,92 @@ def api_device_open_ports():
-def api_get_devices():
-    if not is_authorized():
-        return jsonify({"success": False, "message": "ERROR: Not authorized", "error": "Forbidden"}), 403
+@validate_request(
+    operation_id="get_all_devices",
+    summary="Get All Devices",
+    description="Retrieve a list of all devices in the system.",
+    response_model=DeviceListResponse,
+    tags=["devices"],
+    auth_callable=is_authorized
+)
+def api_get_devices(payload=None):
```

Ruff (0.14.11)

Unused function argument: `payload`

(ARG001)